src/app/project/documents/documents-resolver.service.ts

```diff
@@ -27,7 +27,7 @@ export class DocumentsResolver implements Resolve<Observable<object>> {
     const projectId = route.parent.paramMap.get('projId');
     let currentPage = route.params.currentPage ? route.params.currentPage : 1;
     let pageSize = route.params.pageSize ? route.params.pageSize : 10;
-    let sortBy = route.params.sortBy && route.params.sortBy !== 'null' ? route.params.sortBy : '-datePosted,+displayName';
+    let sortBy = route.params.sortBy && route.params.sortBy !== 'null' || '' ? route.params.sortBy : '-datePosted,+displayName';
```

Comment:
Did you mean:

let sortBy = route.params.sortBy && (route.params.sortBy !== 'null' || route.params.sortBy !== '') ? route.params.sortBy : '-datePosted,+displayName';

??

Reply:
Yes.

Reply:
Further to this comment, by having `route.params.sortBy` it will never by null or ''. You can simply remove those two checks. 

`let sortBy = route.params.sortBy ? route.params.sortBy : '-datePosted,+displayName';` is the same thing.

Reply:
Updated.